pywebsocket3/websocket_server.py

```diff
@@ -157,12 +157,17 @@ def _create_sockets(self):
                         client_cert_ = ssl.CERT_REQUIRED
                 else:
                     client_cert_ = ssl.CERT_NONE
-                socket_ = ssl.wrap_socket(
-                    socket_,
-                    keyfile=server_options.private_key,
-                    certfile=server_options.certificate,
-                    ca_certs=server_options.tls_client_ca,
-                    cert_reqs=client_cert_)
+
+                ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS)
```

Comment:
So far the defaults were used since they were not set when calling ssl.wrap_socket(): server_side=False and ssl_version=PROTOCOL_TLS. Since I am not sure about the implications I used those defaults. For PROTOCOL_TLS it looks like it is deprecated since version 3.10, so going for PROTOCOL_TLS_SERVER seems to be the right choice.